pages/forgot-password.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import * as yup from 'yup';
+import { Button, Input, Navbar } from '@/components/index';
+import HookForm from '@/components/form/Form';
+import { LogoBlack } from '/public/assets/signUp';
+
+const ForgotPassword = () => {
+  const SignInSchema = yup
```

Comment:
This is the forgot password schema (not SignInSchema). Also we can reuse the same schema we used for email. Remind me to discuss about this in our next sync. Cheers.

```suggestion
  const forgotPasswordSchema = yup
```